tests/consensus_tests/test_force_delete_peer.py

```
@@ -0,0 +1,84 @@
+
+    transfers = get_collection_cluster_info(peer_api_uris[0], COLLECTION_NAME)[
+        "shard_transfers"
+    ]
+    assert len(transfers) == 1
+    assert transfers[0]["to"] == list(peer_id_to_url.keys())[-1] # last peer was restarted
+    from_peer_id = transfers[0]["from"]
+
+    # Stop the 'source' node to simulate an unreachable node
+    source_peer_url = peer_id_to_url[from_peer_id]
+    peer_idx = peer_api_uris.index(source_peer_url)
+    p = processes.pop(peer_idx)
```

Ruff (0.8.2)

`processes` may be undefined, or defined from star imports

(F405)